staging/src/k8s.io/client-go/examples/controller-ref-manager/controller/ref_manager.go

```diff
@@ -0,0 +1,126 @@
+package controller
```

Comment:
I believe golint does not like underscore in file names

Reply:
Sorry I don't get this one. You speak about underscore in `ref_manager.go`?

If so, many files in this repo use underscore in file name.
Example:
```
$ ls pkg/controller
...
client_builder.go
controller_ref_manager.go
controller_ref_manager_test.go
controller_utils.go
controller_utils_test.go
...
```

Reply:
true, golint only complains for symbols with underscore.